hack/tools/metrics/main.go

```diff
@@ -107,7 +107,7 @@ func main() {
 	p.Draw(draw.New(canvas))
 	out, err := os.OpenFile(*output, os.O_CREATE|os.O_WRONLY|os.O_TRUNC, os.ModePerm)
 	if err != nil {
-		log.Error(err)
+		log.Fatal(err)
```

Comment:
If we use Fatal, defer function(L58-L62 ) is ignored. so should we return the error? 🤔 
if I made a mistake, I am sorry 🙏

Reply:
Thanks. it will be revised by following:
https://github.com/vdaas/vald/pull/1052/files#r600978617